packages/react-component-library/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+
+import { render, screen } from '@testing-library/react'
+
+import { spacing } from '@royalnavy/design-tokens'
+
+import { ProgressBar } from './ProgressBar'
+import { fixProgressValue } from './fixProgressValue'
+
+describe('Validation', () => {
```

Comment:
I'm wondering if we need to test the implementation detail like this or just add to the `ProgressIndicator` tests.